src/onewire.c

```diff
@@ -173,41 +172,44 @@ static int OWSearch()
 			return false;
 		}
 		// issue the search command
-		xferbyte( OW_SEARCH_ROM );
+		xferbyte(OW_SEARCH_ROM);
 		// loop to do the search
 		do {
 			// read a bit and its complement
 			id_bit = xferbit( true );
 			cmp_id_bit = xferbit( true );
 			// check for no devices on 1-wire
-			if ((id_bit == 1) && (cmp_id_bit == 1))
+			if ((id_bit == 1) && (cmp_id_bit == 1)) {
 				break;
-			else {
+			} else {
 				// all devices coupled have 0 or 1
-				if (id_bit != cmp_id_bit)
+				if (id_bit != cmp_id_bit) {
 					search_direction = id_bit; // bit write value for search
-				else {
+				} else {
 					// if this discrepancy if before the Last Discrepancy
 					// on a previous next then pick the same as last time
-					if (id_bit_number < LastDiscrepancy)
+					if (id_bit_number < LastDiscrepancy) {
 						search_direction = ((ROM_NO[rom_byte_number] & rom_byte_mask) > 0);
-					else
+					} else {
 						// if equal to last pick 1, if not then pick 0
 						search_direction = (id_bit_number == LastDiscrepancy);
+					}
```

Comment:
No it hasn't fallen out, it's still the only statement in there, but all those if statements clearly benefit from having some curly braces added. I had to go check the original Maxim/Dallas app note to verify that it looked equally odd there. :)